clients/ui/frontend/tsconfig.json

```diff
@@ -15,18 +15,12 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
In the dashboard we have [downLevelIteration](https://www.typescriptlang.org/tsconfig/#downlevelIteration) too, cause it can cause some shenanigans with certain tokens when traspilint to old javascript.